website/views/static/plans.jade

```diff
@@ -15,15 +15,15 @@ block content
       p
         =env.t('indivPlan1')
         |&nbsp;
-        a(href='http://habitica.wikia.com/wiki/Guilds' target='_blank')=env.t('guilds')
+        !=env.t('guildsLink')
         = ' ' + env.t('and') + ' '
-        a(href='http://habitica.wikia.com/wiki/Challenges' target='_blank')=env.t('challenges')
+        !env.t('challengesLink')
```

Comment:
*facepalm* Derp
On Tue, Aug 9, 2016 at 6:44 PM Alys <notifications@github.com> wrote:

> In website/views/static/plans.jade
> <https://github.com/HabitRPG/habitrpg/pull/7036#discussion_r74164689>:
>
> >          = ' ' + env.t('and') + ' '
> > -        a(href='http://habitica.wikia.com/wiki/Challenges' target='_blank')=env.t('
>
> challenges')
> > +        !env.t('challengesLink')
>
> This is missing an = after the !, causing the text to appear like this:
> free (or cheap) Guilds and !env.t('challengesLink') can be used
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/7036/files/8cdf35ae9003dd15a0a624164bf80fd4eec15506#r74164689>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALz_8wcw6GGB1h0yo_o5ohrQ6Mkpyzgtks5qeRD6gaJpZM4ICNL2>
> .
>